catalog-info.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: qualified-embed
+  annotations:
+    github.com/project-slug: andela-technology/qualified-embed
+  links:
+    - url: https://www.qualified.io/embedded
+      title: Embed playground
+      icon: url
+  
+  tags:
+    - ror
+    - nodejs
+    - vue
```

I'm not sure what valid tags we have, but the current set doesn't apply. This repo is vanilla JS, with webpack and Playwright tests (not sure if we're tracking those last two).

```suggestion
  tags:
    - javascript
```